util/find-doc-nits

```diff
@@ -173,6 +173,11 @@ sub check()
         if $contents =~ /L<([^>]*)\|([^>]*)>/ && $1 eq $2;
     print "$id Bad =over $1\n"
         if $contents =~ /=over([^ ][^24])/;
+    # Leave out foo'd and foo's
+    print "$id Contraction $1\n"
+        if $contents =~ /([\w]'[abce-rt-z][^'\w])/;
+    #print "$id Maybe contraction \"$1\"\n"
+    #    if $contents =~ /(.{1,10}[\w]'[a-z].{1,10})/;
```

Comment:
I have a small preference to omit the commented-out lines from the committed version, but do not insist.